proof/refine/Invariants_H.thy

```diff
@@ -1289,14 +1055,12 @@ lemma ntfn_splits[split]:
              \<not> P (f2 x2)) \<or>
        (\<exists>x3. ntfn = Structures_H.ntfn.WaitingNtfn x3 \<and>
              \<not> P (f3 x3))))"
-  by (case_tac ntfn; simp)+
+  by (rule ntfn.splits)+
+
 \<comment> \<open>---------------------------------------------------------------------------\<close>
 
 section "Lemmas"
 
-schematic_goal wordBits_def': "wordBits = numeral ?n" (* arch-specific consequence *)
-  by (simp add: wordBits_def word_size)
```

Comment:
If we leave `word_size_def` generic, we can probably leave this one generic as well.

Reply:
From MM discussion: I will try keep it internal as an extra guard rail, even though word_size itself is exported. If things get too bad, we can abort the guard rail.